integration/testnet/network.go

```diff
@@ -721,8 +721,8 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		RewardCut:                    cadence.UFix64(0),
 		CurrentEpochCounter:          cadence.UInt64(epochCounter),
 		NumViewsInEpoch:              cadence.UInt64(epochSetup.FinalView - epochSetup.FirstView - 1),
-		NumViewsInStakingAuction:     cadence.UInt64(100),
-		NumViewsInDKGPhase:           cadence.UInt64(100),
+		NumViewsInStakingAuction:     cadence.UInt64(5),
+		NumViewsInDKGPhase:           cadence.UInt64(60),
```

Comment:
```suggestion
		NumViewsInDKGPhase:           cadence.UInt64(20),
```
Since there are 3 DKG phases, the length of the epoch must be less than `3*dkgPhaseLength + stakingAuctionLength`